components/scanners/cdxgen/README.md

```diff
@@ -0,0 +1,4 @@
+# cdxgen
+
+This component implements a [scanner](https://github.com/smithy-security/smithy/blob/main/sdk/component/component.go)
+that runs CDXGEN and puts the results in a waiting Dependency Track instance. The component does not parse or handle the produced SBOM any other way for now.
```

Comment:
what does a `waiting Dependency Track instance` mean? is it some specific terminology related to DT?

Reply:
it means that we this component won't succeed unless there's a dependency track somewhere ready to receive the sbom